source/grid/grid_in.cc

```diff
@@ -2760,9 +2792,542 @@ GridIn<dim, spacedim>::parse_format (const std::string &format_name)
 template <int dim, int spacedim>
 std::string GridIn<dim, spacedim>::get_format_names ()
 {
-  return "dbmesh|msh|unv|vtk|ucd|xda|netcdf|tecplot";
+  return "dbmesh|msh|unv|vtk|ucd|abaqus|xda|netcdf|tecplot";
 }
 
+namespace
+{
+  template <int dim>
+  class Abaqus_to_UCD
+    {
+    public:
+      Abaqus_to_UCD (void);
+      ~Abaqus_to_UCD (void);
```

Comment:
Just globally replace all `(void)` occurrences in this file by `()`. C++ doesn't require this.